config/environments/production.rb

```diff
@@ -102,6 +102,9 @@
   # why help timing attacks?
   config.middleware.delete(Rack::Runtime)
 
+  require File.expand_path("../../extras/gvl_instrumentation", __FILE__)
+  config.middleware.insert 0, GvlInstrumentation, File.open(Rails.root.join("/srv/lobste.rs/log/production-gvl.log"), "a+")
```

Comment:
I moved the log to its own file. But I don't know if the app has the permission to create it?

Reply:
That'll be fine.